build.sbt

```diff
@@ -23,6 +23,8 @@ lazy val parserCombinatorsVersion = settingKey[String]("")
 lazy val mockitoVersion = "2.20.0"
 lazy val collectionCompatVersion = settingKey[String]("")
 
+disablePlugins(MimaPlugin)
```

Comment:
Ah.  I've just noticed lots of `mimaPreviousArtifacts not set` errors in various upgrades.

I'm guessing this is mitigating that.  That's a regression and I'm sorry.  I'll look to fixing that ASAP.

Reply:
https://github.com/lightbend/mima/issues/328

Reply:
@dwijnand Thanks for jumping in here! We look forward to having your hotfix.

Reply:
Hey.  So I understand better what the change was and what the indended migration was.

If you want you can continue disabling like that, otherwise you can set `mimaFailOnNoPrevious in ThisBuild := false` once and enable it on the projects where you want them (or not at all).

I'm adding that documentation in https://github.com/lightbend/mima/pull/330 (a little hard to read the diff).